backend/app/views/spree/admin/promotions/_form.html.erb

```diff
@@ -33,6 +33,11 @@
         <%= f.check_box :advertise %>
         <%= f.label :advertise %>
       <% end %>
+
+      <%= f.field_container :apply_automatically do %>
+        <%= f.check_box :apply_automatically %>
+        <%= f.label :apply_automatically %>
+      <% end %>
```

Comment:
Honestly, we should create a helper for this. If I had a dollar for every time I've written those four lines... I could purchase Spreecommerce and FirstData...